storage/rocksdb/rdb_comparator.h

```diff
@@ -82,18 +69,17 @@ class Rdb_rev_comparator : public rocksdb::Comparator {
   Rdb_rev_comparator &operator=(const Rdb_rev_comparator &) = delete;
   Rdb_rev_comparator() = default;
 
-  static int bytewise_compare(const rocksdb::Slice &a,
-                              const rocksdb::Slice &b) {
-    return -Rdb_pk_comparator::bytewise_compare(a, b);
-  }
-
   int Compare(const rocksdb::Slice &a, const rocksdb::Slice &b) const override {
-    return -Rdb_pk_comparator::bytewise_compare(a, b);
+    return -a.compare(b);
   }
   const char *Name() const override { return "rev:RocksDB_SE_v3.10"; }
   void FindShortestSeparator(std::string *start,
-                             const rocksdb::Slice &limit) const override {}
-  void FindShortSuccessor(std::string *key) const override {}
+                             const rocksdb::Slice &limit) const override {
+    rocksdb::BytewiseComparator()->FindShortestSeparator(start, limit);
+  }
+  void FindShortSuccessor(std::string *key) const override {
+    rocksdb::BytewiseComparator()->FindShortSuccessor(key);
+  }
```

Comment:
As far as I understand Siying's suggestion is actually a must:  using rocksdb::BytewiseComparator()->FindShortSuccessor() in Rdb_rev_comparator is *not valid*, as "successor" for forward-ordered BytewiseComparator is not a successor for backward-ordered comparator.